Add user guide for calendar

wanguiwaweru opened the pull request:
<!--Read comments, before committing pull request read checklist again

# Checklist:

- I have performed a self-review of my own code,
- I have commented my code, particularly in hard-to-understand areas,
- I have made corresponding changes to the documentation,
- I have added tests that prove my fix is effective or that my feature works,
- New and existing unit tests pass locally with my changes ("bundle exec rake"),
- Title include "WIP" if work is in progress.

-->

Resolves #001 <!--fill issue number-->

### Description
<!-- Please include a summary of the change and which issue is fixed. 
Please also include relevant motivation and context.
Guide questions:
  - What motivated this change (if not already described in an issue)?
  - What alternative solutions did you consider?
  - What are the tradeoffs for your solution?
   
List any dependencies that are required for this change. (gems, js libraries, etc.)

Include anything else we should know about. -->

### Type of change

<!-- Please delete options that are not relevant. -->

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)
* This change requires a documentation update
* Documentation update

### How Has This Been Tested?

<!-- Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce. 
Do we need to do anything else to verify your changes? 
If so, provide instructions (including any relevant configuration) so that we can reproduce? -->

### Screenshots
<!--Optional. Delete if not relevant. 
Include screenshots (before / after) for style changes, highlight edited element.-->

edwinthinks committed:
WIP

edwinthinks committed:
WIP

edwinthinks committed:
Merge branch 'main' into 3260/bank-partner-user-management

edwinthinks committed:
Update with button

edwinthinks committed:
WIP

edwinthinks committed:
WIP

edwinthinks committed:
Update

edwinthinks committed:
Updated

edwinthinks committed:
Add and remove access to partner account for users

edwinthinks committed:
WIP

edwinthinks committed:
WIP

edwinthinks committed:
WIP

edwinthinks committed:
Re-styling

edwinthinks committed:
WIP

edwinthinks committed:
Remove older method for inviting and reseting password for partner

edwinthinks committed:
Fix rubocop complaints

edwinthinks committed:
Merge branch 'main' into 3260/bank-partner-user-management

edwinthinks committed:
Merge branch 'main' into 3260/bank-partner-user-management

edwinthinks committed:
Update styling on table

edwinthinks committed:
Update with partner merge changes

Commit:
Use basic flow instead of turbo

Commit:
fix form errors

awwaiid committed:
Merge branch 'main' into 3260/bank-partner-user-management

 Conflicts:
	Gemfile.lock
	app/controllers/partners_controller.rb
	app/views/partners/show.html.erb

Learningstuff98 committed:
Added pagination for the admin organization index page

Learningstuff98 committed:
Added some tests

Learningstuff98 committed:
Changed the pagination threshold

Learningstuff98 committed:
Merge branch 'main' into pagination-for-organizaation-view

Learningstuff98 committed:
Added some tests

Learningstuff98 committed:
Updated some tests

Learningstuff98 committed:
Updated the request_system_spec

Learningstuff98 committed:
Updated user_system_spec

Learningstuff98 committed:
Updated a test

awwaiid committed:
Merge remote-tracking branch 'origin/main' into 3260/bank-partner-user-management

 Conflicts:
	db/seeds.rb

awwaiid committed:
Satiate Rubocop! [#3260]

edwinthinks committed:
Fix some typos causing reference error

edwinthinks committed:
Prevent reference erro by using current_partner

edwinthinks committed:
Remove outdated test which uses old invite pathway

edwinthinks committed:
Update system test to use new partner/users route

edwinthinks committed:
Fix rubocop

edwinthinks committed:
Merge branch 'main' into 3260/bank-partner-user-management

edwinthinks committed:
Merge branch 'main' into 3260/bank-partner-user-management

edwinthinks committed:
Bring back missing code

edwinthinks committed:
Remove extra locale that doesn't seem used

edwinthinks committed:
Add missing request spec for new controller

edwinthinks committed:
Remove code to prevent broadcasting in model callbacks. Not sure why it was added originally

dorner committed:
Merge branch 'main' into pagination-for-organizaation-view

# Conflicts:
#	db/schema.rb
#	spec/system/admin/organizations_system_spec.rb
#	spec/system/request_system_spec.rb

dorner committed:
Fix spec

dorner committed:
lint fix

dorner committed:
Fix pagination

dorner committed:
Merge branch 'main' into pagination-for-organizaation-view

# Conflicts:
#	app/views/admin/organizations/_list.html.erb

dorner committed:
Fix failing specs

dorner committed:
Spec fixes

dorner committed:
Merge branch 'main' into pagination-for-organizaation-view

# Conflicts:
#	spec/rails_helper.rb
#	spec/system/admin/organizations_system_spec.rb

dorner committed:
Merge branch 'main' into 3260/bank-partner-user-management

# Conflicts:
#	app/controllers/partners_controller.rb
#	app/views/partners/show.html.erb
#	config/routes.rb
#	spec/requests/partners_requests_spec.rb
#	spec/system/partner_system_spec.rb

dorner committed:
Lint + spec fix

dorner committed:
Fix

dorner committed:
Merge branch 'main' into 3260/bank-partner-user-management

# Conflicts:
#	config/locales/en.yml

dorner committed:
CR fixes

dorner committed:
Fix spec

dorner committed:
Show inventory history for pre-event read

dependabot[bot] committed:
build(deps): bump jwt from 2.8.1 to 2.8.2

Bumps [jwt](https://github.com/jwt/ruby-jwt) from 2.8.1 to 2.8.2.
- [Release notes](https://github.com/jwt/ruby-jwt/releases)
- [Changelog](https://github.com/jwt/ruby-jwt/blob/main/CHANGELOG.md)
- [Commits](https://github.com/jwt/ruby-jwt/compare/v2.8.1...v2.8.2)

---
updated-dependencies:
- dependency-name: jwt
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump bugsnag from 6.27.0 to 6.27.1

Bumps [bugsnag](https://github.com/bugsnag/bugsnag-ruby) from 6.27.0 to 6.27.1.
- [Release notes](https://github.com/bugsnag/bugsnag-ruby/releases)
- [Changelog](https://github.com/bugsnag/bugsnag-ruby/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bugsnag/bugsnag-ruby/compare/v6.27.0...v6.27.1)

---
updated-dependencies:
- dependency-name: bugsnag
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

awwaiid committed:
Merge pull request #3266 from rubyforgood/3260/bank-partner-user-management

[3260] Add feature to manage partner users for a Partner as a Organization/Bank

awwaiid committed:
Merge remote-tracking branch 'origin/main' into pagination-for-organizaation-view

awwaiid committed:
Merge pull request #3490 from rubyforgood/pagination-for-organizaation-view

Pagination for organization view

dorner committed:
Update dependabot.yml

AweysAhmed committed:
edited css file to make message less omnious

dependabot[bot] committed:
build(deps-dev): bump pry-rails from 0.3.9 to 0.3.11

Bumps [pry-rails](https://github.com/rweng/pry-rails) from 0.3.9 to 0.3.11.
- [Commits](https://github.com/rweng/pry-rails/compare/v0.3.9...v0.3.11)

---
updated-dependencies:
- dependency-name: pry-rails
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rspec-rails from 6.1.2 to 6.1.3

Bumps [rspec-rails](https://github.com/rspec/rspec-rails) from 6.1.2 to 6.1.3.
- [Changelog](https://github.com/rspec/rspec-rails/blob/main/Changelog.md)
- [Commits](https://github.com/rspec/rspec-rails/compare/v6.1.2...v6.1.3)

---
updated-dependencies:
- dependency-name: rspec-rails
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump knapsack_pro from 7.6.0 to 7.6.1

Bumps [knapsack_pro](https://github.com/KnapsackPro/knapsack_pro-ruby) from 7.6.0 to 7.6.1.
- [Changelog](https://github.com/KnapsackPro/knapsack_pro-ruby/blob/master/CHANGELOG.md)
- [Commits](https://github.com/KnapsackPro/knapsack_pro-ruby/compare/v7.6.0...v7.6.1)

---
updated-dependencies:
- dependency-name: knapsack_pro
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

napster235 committed:
Show column if request organization or request partner has default_storage_location

napster235 committed:
Add specs

cielf committed:
Merge pull request #4461 from rubyforgood/dependabot-monthly

Update dependabot.yml

napster235 committed:
Enhance specs

jimmyli97 committed:
Fix #4447 Fix #3987 Fix #3755 merge README into CONTRIBUTING and document deploy process (#4449)

* Fix dead link and typos in sbv

* Merge README slack and welcome info into CONTRIBUTING

* Merge README env setup into CONTRIBUTING

* Merge README contributing overview into CONTRIBUTING

* Merge README issue guidelines into CONTRIBUTING

* Merge README squash guidelines into CONTRIBUTING

* Merge README PR guidelines into CONTRIBUTING

* Move CONTRIBUTING PR steps into workflow

* Merge README test guideliens into CONTRIBUTING

* Merge README deployment into CONTRIBUTING

* Merge README additional notes into CONTRIBUTING

* Merge README become contributor into CONTRIBUTING

* Merge README debugging into CONTRIBUTING

* Fix #4447 add note about automated tests

* Fix #3987 address #3755 document prod branch and date versioning

* Move deploy process to README

dorner committed:
Fix totals

vsrnth committed:
4310 - Update item quantity validation error message (#4412)

* Updates Item validation error message

* Updates unit tests

* Updated related system specs

* Updated validation message

- Updated validation message
- Updated related unit and system specs

* Fixes system spec

* Adds a custom validation to handle edge cases

- Updates system specs and adds required unit tests

* Fixes bug in unit tests

napster235 committed:
Remove the POST /partner/ID/invite_partner_user route (#4467)

danielabar committed:
3090 v2 distribution confirmation (#4367)

* [#3090] Use a Stimulus controller to intercept distribution form submit
and render a bootstrap modal displaying partner name, storage location,
and list of items and quantities. The No button simply closes the modal
so user can return to the new distribution form to make further edits.
The Yes button submits the form and the existing workflow continues.

* [#3090] Maintain distribution system tests wrt confirmation modal

* [#3090] Verify items and quantities in distribution confirmation modal

* [#3090] Fix request system test wrt distribution confirmation modal

* [#3090] Fix distribution system tests wrt instance var and new path

* [#3090] PR feedback: update wording on confirmation modal

* [#3090] Integrate validation into confirmation:
1. Add validation only endpoint for distribution
2. Update confirmation JS controller to check if form is valid
(using the new validation endpoint) before displaying modal.

* [#3090] PR feedback:
Combine same named items & quantities for display in confirmation modal

* [#3090] PR feedback:
* Move confirm style into custom
* Remove call to debug form data
* Remove console logs for valid and invalid

* [#3090] PR feedback:
* Move modal content generation to server side rendering
* Remove unneeded dom targets from stimulus controller

* [#3090] PR feedback: use specific partner and storage names in system test

* [#3090] PR feedback: rename to confirmation controller

* [#3090] PR feedback: use meta level authenticity token

* [#3090] PR feedback: Quotes for consistency in fetch request headers

AweysAhmed committed:
Merge branch 'main' into 4242-update-colour-for-inactive-items-message

elasticspoon committed:
feat: replace org_user role deactivate with removal

The issue was that banks could "deactivate" a user
(which is essentially a soft delete of the whole User record).
However, if that user then went to a Partner they would not be able to invite
that user because that user would exist (new User cannot be created)
but be soft deleted (existing User is treated like they didn't have permissions).

We changed the logic around deactivation to address the role not the user.

I attempted to implement logic around deactivating roles themselves
in #4392, #4382. But the logic became very complex.

We decided (#3587 (comment)) to go for a more architerurally simple solution
of removing the deactivation functionality and just deleting the roles.

To migrate the existing data I decided to reactivate any discarded user
with the org user role and remove the org user role from them. (My guess
is that those are the org users that actually got deactivated)

napster235 committed:
[4409] - Rephrase context

napster235 committed:
[4409] - Switch or clause to firstly get partner location if present

noah-puckett committed:
remove delete button from user list

noah-puckett committed:
remove test for user deletion

noah-puckett committed:
optionally remove user delete functionality from controller

norrismei committed:
Wrap non-footer elements in bounding box (#4479)

Naraveni committed:
4407- Showing All Item Fields In Item View (#4431)

napster235 committed:
Check for the new named header

napster235 committed:
Fix failing specs

elasticspoon committed:
fix: review comments

also simplification/correction of some test logic

elasticspoon committed:
fix: simplify migration

All discarded users are org_user meaning they have been deactivated. To
simplify everything we will just remove all thier roles.

napster235 committed:
Fix request system specs

cielf committed:
Merge pull request #4480 from noah-puckett/4434-remove-superuser-delete-functionality

4434 remove superuser delete functionality

cielf committed:
Merge pull request #4466 from napster235/4409-only_show_fulfillment_column_if_partner_has_default_location

[4409] Only show fulfillment column if partner has default location

AweysAhmed committed:
Merge branch 'main' into 4242-update-colour-for-inactive-items-message

clair13 committed:
Correct word for the NDBN report

AweysAhmed committed:
updated message to indicate low priority of message

elasticspoon committed:
fix: remove mini_racer to prevent segfaults (and because we don't use it) (#4478)

* fix: downgrade mini_racer to prevent segfaults

Currently when I run `rails assets:precompile` the application
segfaults. This seems to be related to mini_racer.

The easiest solution is to downgrade to a version of mini_rancer without
these issues.

See https://github.com/rubyjs/mini_racer/issues/300 for more info.

* fix: remove it instead

cielf committed:
Merge pull request #4462 from AweysAhmed/4242-update-colour-for-inactive-items-message

Fixes #4242 so that  css file doesn't make the warning message look scary

cielf committed:
Merge pull request #4493 from clair13/4470-wording-correction-for-the-NDBN-report

Correct word for the NDBN report

jimmyli97 committed:
Fix #4311 add comment field to product drive participant  (#4442)

* Fix #4311 add comment to product drive participant

* Can add comments when creating new participant under Community
* Can add comments when creating new participant under Donation
* Can edit comments when editing participant under Community
* Can view comments on single participant page after clicking "View"
  action on Product Drive Particpants index
* Validates comment to be less than 501 characters
* RSpec tests written for all above changes

* REFACTOR rename form button and comments to be consistent

* Database migration spec change

* Use request test, not system, for comment display

* Hard code comment in comment rspec

cielf committed:
Merge pull request #4477 from elasticspoon/feat/replace-role-deactivation-with-removal

replace org_user deactivation with removal of org_user role

dependabot[bot] committed:
build(deps): bump newrelic_rpm from 9.10.2 to 9.11.0 (#4490)

Bumps [newrelic_rpm](https://github.com/newrelic/newrelic-ruby-agent) from 9.10.2 to 9.11.0.
- [Release notes](https://github.com/newrelic/newrelic-ruby-agent/releases)
- [Changelog](https://github.com/newrelic/newrelic-ruby-agent/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/newrelic/newrelic-ruby-agent/compare/9.10.2...9.11.0)

---
updated-dependencies:
- dependency-name: newrelic_rpm
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump strong_migrations from 1.8.0 to 2.0.0 (#4491)

Bumps [strong_migrations](https://github.com/ankane/strong_migrations) from 1.8.0 to 2.0.0.
- [Changelog](https://github.com/ankane/strong_migrations/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ankane/strong_migrations/compare/v1.8.0...v2.0.0)

---
updated-dependencies:
- dependency-name: strong_migrations
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump standard from 1.37.0 to 1.39.0 (#4492)

Bumps [standard](https://github.com/standardrb/standard) from 1.37.0 to 1.39.0.
- [Release notes](https://github.com/standardrb/standard/releases)
- [Changelog](https://github.com/standardrb/standard/blob/main/CHANGELOG.md)
- [Commits](https://github.com/standardrb/standard/compare/v1.37.0...v1.39.0)

---
updated-dependencies:
- dependency-name: standard
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop-rails from 2.25.0 to 2.25.1 (#4488)

Bumps [rubocop-rails](https://github.com/rubocop/rubocop-rails) from 2.25.0 to 2.25.1.
- [Release notes](https://github.com/rubocop/rubocop-rails/releases)
- [Changelog](https://github.com/rubocop/rubocop-rails/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-rails/compare/v2.25.0...v2.25.1)

---
updated-dependencies:
- dependency-name: rubocop-rails
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump knapsack_pro from 7.6.1 to 7.6.2 (#4489)

Bumps [knapsack_pro](https://github.com/KnapsackPro/knapsack_pro-ruby) from 7.6.1 to 7.6.2.
- [Changelog](https://github.com/KnapsackPro/knapsack_pro-ruby/blob/master/CHANGELOG.md)
- [Commits](https://github.com/KnapsackPro/knapsack_pro-ruby/compare/v7.6.1...v7.6.2)

---
updated-dependencies:
- dependency-name: knapsack_pro
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump standard from 1.39.0 to 1.39.1 (#4497)

Bumps [standard](https://github.com/standardrb/standard) from 1.39.0 to 1.39.1.
- [Release notes](https://github.com/standardrb/standard/releases)
- [Changelog](https://github.com/standardrb/standard/blob/main/CHANGELOG.md)
- [Commits](https://github.com/standardrb/standard/compare/v1.39.0...v1.39.1)

---
updated-dependencies:
- dependency-name: standard
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

elasticspoon committed:
feat: improve child functionality

Co-authored-by: isidzukuri <axesigon@gmail.com>

elasticspoon committed:
fix: mix strong migrations (#4498)

Version 10 no longer supported. Updating to 12.

dorner committed:
Speed up item page

dorner committed:
lint fix

dorner committed:
Only use active kits

dorner committed:
Better fix for active

cielf committed:
Merge pull request #4499 from rubyforgood/item-index-caching

Speed up item page

grantmca committed:
Added some logic to the seed script to populate half of the times from Pawnee Diaper Bank Org with a on_hand_minimum_quantity and a on_hand_recommended_quantity for demo and testing purposes

awwaiid committed:
Merge pull request #4469 from rubyforgood/4433-inventory-history

Show inventory history for pre-event read

AweysAhmed committed:
rearranged the ui to alter the flow to start with storage locations

AweysAhmed committed:
Merge branch 'main' into 4474-switch-steps_for_bank_onboarding

AweysAhmed committed:
fixed failing spec and updated comment

grantmca committed:
Update the seed script to grab half of the items with the lowest total quantity and for each one assign a random minimum and recommended value based off the max and min values in the total quantity

elasticspoon committed:
fix: review comments

fix seen to ensure only requestable items seeded
fix item to items

AweysAhmed committed:
fixed ordering of the spec file

cielf committed:
Update dashboard_system_spec.rb

Minor wording change in comment

cielf committed:
Merge pull request #4507 from AweysAhmed/4474-switch-steps_for_bank_onboarding

Fixes-4474-by-switching-onboarding-logic-in-ui

cielf committed:
Merge pull request #4500 from grantmca/4475-add-min-recomended-to-seed

Enhancing Seed Script with Minimum and Recommended Quantities

cielf committed:
Merge pull request #4495 from rubyforgood/feat/improve-child-functionality

Improve child functionality by allowing multiple items to be requested

awwaiid committed:
Fix a Rails 7.2 deprecation (#4506)

DEPRECATION WARNING: Passing the class as positional argument is deprecated and will be removed in Rails 7.2.

elasticspoon committed:
feat: improve bin/setup (#4494)

* feat: improve bin/setup

* fix: change font color

awwaiid committed:
Update prod db fetch to not use the DB redundantly (#4505)

Without this it was connecting to the database which prevented it from
doing the drop/create. This also removes a now-unneeded work-around for
that issue

Naraveni committed:
Button Not getting Activated In New Distribution

Naraveni committed:
Merge branch 'main' into 4438-ACTIVATE-ADD-ANOTHER-ITEM-AFTER-ERROR

cielf committed:
Merge pull request #4485 from Naraveni/4438-ACTIVATE-ADD-ANOTHER-ITEM-AFTER-ERROR

4438 - Button Remain Activated After Selecting the Storage

awwaiid committed:
Remove implicit rails_helper require (#4529)

awwaiid committed:
Configure bank organization units (#4432)

* WIP Added Custom request units at ogranizational level

* Add unique validation to custom organization unit name

* Added flipper conditionals and specs

* Fixed the button name and delete issue

* Updated to allow organizations to delete units

* Clean out some lint!

* Use select2 in tags-mode for Bank unit configuration

* Modify select2 to allow multiple instances on the same page

* Update spec to match request units param

* Hide the select2 drop-down for units [#4396]

* Use unit-names instead of id's when configuring bank [#4396]

* Update some org-unit config specs [#4396]

* Remove outdated comment [#4396]

* Use a dup of params and don't mutate the original [#4396]

* Remove implicit rails_helper require [#4396]

* Explicitly check for packs-disabled case [#4396]

* Move most organization system specs to request spec [#4396]

* Clean out some lint [#4396]

* Switch two more unit-name validation consts [#4396]

---------

Co-authored-by: Liz Catoe <liz.catoe@priviahealth.com>
Co-authored-by: srinikotla <38926870+srinikotla@users.noreply.github.com>

elasticspoon committed:
fix: fix migration for prod data (#4528)

Prod had some bad data that caused the migration to fail since it would
attempt to create requested items where those items no longer existed.

jimmyli97 committed:
RED add rspec to test editing distribution when storage location has total inventory 0

jimmyli97 committed:
GREEN fix #4537 can edit original storage location when total inventory is 0

cielf committed:
Merge pull request #4538 from jimmyli97/4537-fix-storage-location

Resolves #4537 fix storage location when total inventory is 0

wanguiwaweru committed:
Add User Guide for the Calendar

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+Click on the "Pick Ups & Deliveries" section to view the scheduled calendar events. 
+
+![PickUp & Delivery](images\pickup&delivery.jpg)
```
The slash in "images\pickup&delivery.jpg" should be a forward slash.  Are you perhaps working on a Windows machine?

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+The Distribution Schedule page shows details including the associated Partner,the Time of Distribution, Source Inventory, the Total number of items, and the status of the distribution.
+
+![Specific Day Distribution](images\specific_day_distribution_schedule.jpg)
```
Similarly here, forward slash please.   

I need to write up a short style guide for everyone for this...
Partner should be capitalized, but time, distribution, source, inventory, total should be lower case.

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+Click "View" for more details on the Source location, the Agency representative, the delivery method, the Shipping cost, comments and the State of the distribution.
+
+![Distribution from Source Inventory to Partner](images\distribution_from_source_to_partner.jpg)
```
Similarly here, forward slash please.  
Please make agency, shipping  lower case
Should  be 'status' instead of 'state',   And "storage, not "Source"

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+![Specific Day Distribution](images\specific_day_distribution_schedule.jpg)
+
+If the distribution has been done, you can mark it as complete by clicking on "Distribution Complete".
```
Does marking it complete remove it from the calendar?    If so, please indicate that.  (It might be obvious in the video, but it isn't in text).

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+![Distribution from Source Inventory to Partner](images\distribution_from_source_to_partner.jpg)
+ 
+Click "Print" to print the details of the distribution, for instance if you need to provide the list for those in the warehouse to pack up. 
```
Actually, we now allow them to print the requests for the "pick lists" for warehouse use.      Reasons to print the details  include providing it as a contents list or receipt.

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+Click on "Copy Calendar URL to clipboard" to get the Calendar URL to be added to Google Calendar. On Google calendar, in the "Other Calendars" section click on the "+" which will bring up several choices.
+
+![Other Calendars](images\Other_calendars.jpg)
```
Similarly here, forward slash please.  Also, the convention we're using is lower case snake case for file names (so other_calendars.jpg)

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+## Sync Calendar to Google Calendar
+
+You can sync the App calendar to Google calendar for more convinience.
```
should be convenience, not convinience.   I would also say "Pick Up and Deliveries Calendar", rather than "App calendar".

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
-Not yet written
\ No newline at end of file
+# Calendar
```
Let's use "Pickup & Deliveries Calendar" as the title here.  It is the only calendar we have,  but the business reason for being here is to look at pickups and deliveries.      (Though I do see we are not being consistent within the app -- the title in the menu and on the view do not match.  I'll make a note to address that.)

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+The calendar shows the Pickup and delivery schedule. It shows past and upcoming distributions in a given month.
+
+Click on the "Pick Ups & Deliveries" section to view the scheduled calendar events. 
```
I would say 'distributions' instead of 'calendar events' here.   This is just phrasing things more in terms of what is happening for the business.

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+![PickUp & Delivery](images\pickup&delivery.jpg)
+
+Click on a specific event to view more details about the scheduled event. 
```
I would say "Click on any scheduled distribution to view more details about it."

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+You can sync the App calendar to Google calendar for more convinience.
+
+Click on "Copy Calendar URL to clipboard" to get the Calendar URL to be added to Google Calendar. On Google calendar, in the "Other Calendars" section click on the "+" which will bring up several choices.
```
the second "Calendar" should be lower case.  The one in 'Google calendar' should be upper case in both cases (because Google Calendar is the proper name of the application.)

cielf commented on doc/user_guide/bank/essentials_calendar.md:
```diff
@@ -1 +1,35 @@
+
+![Add Calendar](images\Add_Calendar.jpg)
+
+Events from the App Calendar should be accessible to you on Google Calendar when the sync is complete.
```
Instead of App Calendar I would say "Human Essentials Pickup and Deliveries Calendar"

cielf reviewed (changes requested):
Hey @wanguiwaweru.  Thank you very much for taking this on!  

Overall - a good start!  Fair warning, though -- I am a really hard editor.   So there are a a lot of small things on this pass.   (I'm also thinking a bit as I write this - so it's a longer review than it should be).  

You're also the first one through the process -- it looks like I'll need to provide some guidance about things like formats of file names, and capitalization up front for everyone else.  

I'm going to introduce a directory structure for the images too -- there's just going to be too many to keep track of otherwise, and it will free us from some of the need for the naming convention I've been using.  But don't worry about that right now.

A couple of general pieces of advice for the next section you take on:

1/  When you are transcribing from the video, check if the terms you think you hear are the terms we are using in the app.    
2/  Think about what the business is trying to do, rather than what the steps are.   (Why do they want to press that button?)  This is very hard when you're new to the system, as you just don't  *know*,  but it will help you learn the system, and it will get easier with experience.

Thanks again!

cielf commented:
We're going to have to figure out how to get this merged into the user guide branch instead of main as well.

grantmca committed:
Refactor `schedule` action in `distributions_controller` for better performance (#4509)

- Separated the handling of JSON and HTML requests. The instance variable is not used in the HTML view, eliminating unnecessary data loading.
- Implemented JSON-specific requests to leverage the `start` and `end` parameters from the FullCalendar library. This restrict tags the fetched distributions to the specified dates instead of all historical records.
- Resolved the N+1 query issue by eager loading the associated partners with the distributions.
- Add an index on issued_at to speed up the filtering on distributions

mdphillips375 committed:
4481 Print Individual Donation Receipts (#4484)

* 4481 Print Individual Donation Receipts

* 4481 Print Individual Donations add zero dollars and tests

* 4481  Print  Individual Donation Receipts Requested Changes

* 4481 Print Individual Donation PDFs hardcode all test parameters

awwaiid committed:
Downgrade strong migrations since our prod db is pg v10

awwaiid committed:
Add note on pinning strong_migrations

awwaiid committed:
Merge pull request #4545 from rubyforgood/downgrade-strong-migrations

Downgrade strong migrations since our prod db is pg v10

wanguiwaweru committed:
Update calendar user guide

cielf committed:
Merge pull request #4460 from cielf/user_documentation_outline

First steps toward bank user guide

wanguiwaweru committed:
Merge branch 'main' of https://github.com/wanguiwaweru/human-essentials into add-user-guide-for-calendar
